INSTRUCTION.md

```diff
@@ -0,0 +1,15 @@
+#Dockerhub
+repository https://hub.docker.com/repository/docker/deadmarvald/todoapp/
+
+
+#Build container
+docker build -t todoapp:1.0.0 .
```

Comment:
Ensure that the Dockerfile is in the current directory when running the 'docker build' command, as the '.' at the end specifies the build context.